content/en/containers/kubernetes/installation.md

```diff
@@ -6,60 +6,61 @@ aliases:
-      text: 'Collect your application logs'
-    - link: '/agent/kubernetes/tag'
-      tag: 'Documentation'
-      text: 'Assign tags to all data emitted by a container, Pod, or Node'
+      text: 'Further Configure the Datadog Agent on Kubernetes'
+    - link: 'https://github.com/DataDog/helm-charts/blob/main/charts/datadog/README.md#all-configuration-options'
+      tag: 'GitHub'
+      text: 'Datadog Helm chart - All configuration options'
+    - link: 'https://github.com/DataDog/helm-charts/blob/main/charts/datadog/README.md#upgrading'
+      tag: 'GitHub'
+      text: 'Upgrading Datadog Helm'
 ---
 
-## Installation
+## Overview
+
+This page provides instructions on installing the Datadog Agent in a Kubernetes environment. By default, the Datadog Agent runs in a DaemonSet.
 
-This page provides instructions on installing the Datadog Agent in a Kubernetes environment. For dedicated documentation and examples for major Kubernetes distributions including AWS Elastic Kubernetes Service (EKS), Azure Kubernetes Service (AKS), Google Kubernetes Engine (GKE), Red Hat OpenShift, Rancher, and Oracle Container Engine for Kubernetes (OKE), see [Kubernetes distributions][1].
+For dedicated documentation and examples for major Kubernetes distributions including AWS Elastic Kubernetes Service (EKS), Azure Kubernetes Service (AKS), Google Kubernetes Engine (GKE), Red Hat OpenShift, Rancher, and Oracle Container Engine for Kubernetes (OKE), see [Kubernetes distributions][1].
 
 For dedicated documentation and examples for monitoring the Kubernetes control plane, see [Kubernetes control plane monitoring][2].
 
-### Minimum Agent and Cluster Agent versions
+### Minimum Kubernetes and Datadog Agent versions
 
 Some features related to later Kubernetes versions require a minimum Datadog Agent version.
 
-| Kubernetes version | Agent version  | Cluster Agent version | Reason                                |
-|--------------------|----------------|-----------------------|---------------------------------------|
-| 1.16.0+            | 7.19.0+        | 1.9.0+                | Kubelet metrics deprecation           |
-| 1.21.0+            | 7.36.0+        | 1.20.0+               | Kubernetes resource deprecation       |
-| 1.22.0+            | 7.37.0+        | 7.37.0+               | Support dynamic service account token |
+| Kubernetes version | Agent version  | Reason                                |
+|--------------------|----------------|---------------------------------------|
+| 1.16.0+            | 7.19.0+        | Kubelet metrics deprecation           |
+| 1.21.0+            | 7.36.0+        |  Kubernetes resource deprecation       |
+| 1.22.0+            | 7.37.0+        |  Support dynamic service account token |
+
+See also: [Minimum Kubernetes and Cluster Agent versions][8].
+
+## Installation
+
+You have the following options for installing the Datadog Agent on Kubernetes:
+
+- The [Datadog Operator][9], a [Kubernetes Operator][10] (Recommended)
```

Comment:
Is this really accurate today? I recall Helm being our preferred method of installation.

Reply:
nope - our preferred method of installation is Datadog Operator, now that it's GA